cmd/monaco/integrationtest/integration_test_utils.go

```diff
@@ -33,9 +33,25 @@ import (
 )
 
 func CreateDynatraceClients(t *testing.T, environment manifest.EnvironmentDefinition) *client.ClientSet {
-	clients, err := dynatrace.CreateClientSet(environment.URL.Value, environment.Auth)
+	var clients *client.ClientSet
+	var err error
+	if environment.Auth.OAuth == nil {
+		clients, err = client.CreateClassicClientSet(environment.URL.Value, environment.Auth.Token.Value, client.ClientOptions{
+			SupportArchive:  support.SupportArchive,
+			CachingDisabled: true,
```

Comment:
Please mention somewhere (e.g. the commit message ;) ) WHY caching is disabled for E2E tests. This seems like a prime candidate for forgetting about stumbling over again in the future